sources/Licenses/README.md

```diff
@@ -0,0 +1,31 @@
+## Motivation
+
+Licenses, such as those from [Creative Commons](https://creativecommons.org/licenses/), are often miswritten and mismatches between version statements (2.0 or v2) are not easy to track. The Licenses.yml rule aims to comprehensively solve these problems.
+
+We preferred to adopt the list of Debian-compatible licenses due to the fact that Debian, as a reliable organization, is also relatively selective in the licenses it accepts. Furthermore, the information provided with the [Debian Free Software Guidelines](https://wiki.debian.org/DebianFreeSoftwareGuidelines) (DFSG) is accessible and permissively licensed.
+
+## Methodology
+
+1. We create a basic and regex capable list of phrases from the DFSG-compatible Licenses
+2. Separated licences from GNU and Creative Commons under their own headings
+3. Added other BSD and Creative Commons licenses
+4. Removed licenses that are too short or use common words, such as Artistic License 1.0
```

Comment:
I think this one should be in the Licenses.yml file, but commented with a note

Reply:
I don't think that's necessary, since unlike Apache, it's very rare.